android/app/google-services.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
check this https://stackoverflow.com/questions/34251441/commiting-google-services-json-to-github

it's okay, tho if you set firebase settings for testing. but it's a hazard in general to commit google-services.json.